CodeReview/src/items/VReviewComment.cpp

```diff
@@ -36,25 +36,60 @@ namespace CodeReview
 
 DEFINE_ITEM_COMMON(VReviewComment, "item")
 
-VReviewComment::VReviewComment(Visualization::Item* parent, NodeType* node, const StyleType* style)
-	: Super{parent, node, style}
+VReviewComment::VReviewComment(Visualization::Item* parent, NodeType* nodeType, const StyleType* style)
```

Comment:
Why did you rename this? `node` is correct. What's passed in is not a type but a node whose type is `NodeType`.